pkg/lib/cockpit.js

```diff
@@ -2524,7 +2524,7 @@ function factory() {
                 href = href.substr(1);
 
             const pos = href.indexOf('?');
-            let first = href;
+            let first;
             if (pos === -1)
                 first = href;
```

Comment:
This is better written as

    const first = (pos === -1) ? href : href.substr(0, pos);